homework_11/homework_11.py

```diff
@@ -0,0 +1,78 @@
+# # 1 Написати власний декоратор, задачею якого має бути друк назви функції і
+# # часу, коли вона була викликана. Декоратор має працювати для різних функцій однаково.
+# import time
+
+
+# def deco_time_and_name(func_to_deco):
+#     def print_time_and_name():
+#         str_name = func_to_deco.__name__
+#         print(f"function name - {str_name}")
+#         str_time = time.ctime()
+#         print(f"start function time - {str_time}")
+#         func_to_deco()
```

Comment:
Use `return` statement here to be able to decorate functions that return some value